src/notification-banner/notification-banner.styles.tsx

```diff
@@ -94,3 +100,28 @@ export const StyledIcon = styled(CrossIcon)`
     width: 1.875rem;
     color: ${Color.Neutral[8]};
 `;
+
+export const ContentWrapper = styled.div<ContentWrapperStyleProps>`
+    ${(props) => {
+        const gradient =
+            "linear-gradient(to bottom, black 50%, transparent 100%)";
+        if (props.$collapsedHeight)
+            return css`
+                height: ${props.$collapsedHeight}px;
```

Comment:
should this be a `max-height` instead? if content is short, there will be empty space